oktapam/data_source_database_password_settings.go

```diff
@@ -0,0 +1,93 @@
+package oktapam
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/okta/terraform-provider-oktapam/oktapam/client"
+	"github.com/okta/terraform-provider-oktapam/oktapam/client/wrappers"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/attributes"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/descriptions"
+)
+
+func dataSourceDatabasePasswordSettings() *schema.Resource {
+	return &schema.Resource{
+		Description: descriptions.SourcePasswordSettings,
```

Comment:
This should be SourceDatabasePasswordSettings

Reply:
Pushed a commit to add the new one and update the existing to specify it's for servers.